templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -527,8 +527,9 @@ Parameters:
     Type: String
   KeyPairName:
     ConstraintDescription: Must be the name of an existing EC2 Key Pair.
-    Description: The EC2 Key Pair to allow SSH access to the instances
-    Type: AWS::EC2::KeyPair::KeyName
+    Description: The EC2 Key Pair to allow SSH access to the instances. If empty the VPC default key will be used.
```

Comment:
`If empty the VPC default key will be used`

`If empty the key pair from Atlassian Standard Infrastructure stack will be used.` It is probably mouthful but it cannot use _any_ key from the VPC, it needs to be ASI?